index.ts

```diff
@@ -149,20 +148,16 @@ const fetchOmnivore = async (
         }
 
         if (highlights?.length) {
-          const highlightBlock = await logseq.Editor.insertBlock(
-            articleBlock.uuid,
-            highlightTitle
-          )
-          if (!highlightBlock) {
-            throw new Error('block error')
-          }
-
-          for (const highlight of highlights) {
-            await logseq.Editor.insertBlock(
-              highlightBlock.uuid,
-              highlight.quote
-            )
-          }
+          const highlightBatch = highlights.map(it => {
+            let noteChild = it.annotation ? { content: 'My Notes: ' + it.annotation } : undefined
```

Comment:
can we use es6?

```
let noteChild = it.annotation ? { content: `My Notes: ${it.annotation : undefined}` }
```

Reply:
it doesn't seem like `noteChild`  being reassigned later, I believe we can use `const` here.

Reply:
@rupin90 thanks! You are right, i was thinking I'd have to mutate that. Fixing both!